rllib/core/learner/tf/tf_learner.py

```diff
@@ -349,6 +347,25 @@ def remove_module(self, module_id: ModuleID) -> None:
         if self._enable_tf_function:
             self._update_fn = tf.function(self._do_update_fn, reduce_retracing=True)
 
+    def _make_distributed_strategy(self):
+        """Create a distributed strategy for the learner."""
```

Comment:
Same, can you add a little more explanation here on what a "strategy" is and which types exist (an example?)?

Reply:
Strategy is a tf distributed strategy object. 

The different types of strategies are contained within the function.